avocado/utils/disk.py

```
@@ -102,12 +102,27 @@ def delete_loop_device(device):
+    """
+    Reads mentioned number of blocks from device to destination
+
+    :param disk: disk absolute path
+    :param read_record: Numbers of blocks to read from disk
+    :param out_file: Destination file
+    :rtype: boolean
+    """
+    cmd = f"dd count={read_records} if={disk} of={out_file}"
+    if process.system(cmd, ignore_status=True, sudo=True) != 0:
+        return False
+    return True
```

Ruff (0.12.2)

Return the negated condition directly

Inline condition

(SIM103)